loader/icd.c

```diff
@@ -26,13 +26,16 @@
 #include <string.h>
 
 KHRicdVendor *khrIcdVendors = NULL;
+char *khrEnableTrace = NULL;
```

Comment:
I would use an integer here to store the state of the tracing activation.
The string returned by getenv should not change through the execution, so better compare it early.

Reply:
Fixed. I defined it as integer.